.github/workflows/ci.yml

```diff
@@ -32,6 +32,8 @@ jobs:
         run: "mount | grep cgroup"
       - uses: actions/checkout@v3
       - uses: actions-rust-lang/setup-rust-toolchain@v1
+        env:
```

Comment:
Shouldn't this be `env-vars` according to this [doc](https://github.com/marketplace/actions/rust-cache)